src/com/jedk1/jedcore/listener/AbilityListener.java

```diff
@@ -393,7 +393,7 @@ public void onPlayerSneak(PlayerToggleSneakEvent event) {
 				if (abil.equalsIgnoreCase("drain")) {
 					new Drain(player);
 				}
-				if (abil.equalsIgnoreCase("wakefishing")) {
+				if (abil.equalsIgnoreCase("wakefishing") && !CoreAbility.hasAbility(event.getPlayer(), WakeFishing.class)) {
```

Comment:
I added this because a player was able to stack this move somehow with a toggle sneak mod, and it just poured out stacks and stacks of fish a second